utils/remote_control/navigator_emergency_control/nodes/nodes/navigator_emergency.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+
+from remote_control_lib import RemoteControl
+import rospy
+from sensor_msgs.msg import Joy
+
+
+__maintainer__ = "Anthony Olive"
+__email__ = "anthony@iris-systems.net"
+__copyright__ = "Copyright 2016, MIL"
+__license__ = "MIT"
+
+
+rospy.init_node("emergency")
+
+
+class Joystick(object):
+
+    def __init__(self):
+        self.force_scale = rospy.get_param("~force_scale", 600)
+        self.torque_scale = rospy.get_param("~torque_scale", 500)
+
+        self.remote = RemoteControl("emergency", "/wrench/emergency")
+        rospy.Subscriber("joy_emergency", Joy, self.joy_recieved)
+
+        self.active = False
+        self.reset()
+
+    def reset(self):
+        '''
+        Used to reset the state of the controller. Sometimes when it
+        disconnects then comes back online, the settings are all out of whack.
+        '''
+        self.last_kill = False
+        self.last_station_hold_state = False
+        self.last_auto_control = False
+        self.last_emergency_control = False
+        self.last_shooter_load = False
```

Comment:
Can you add shooter_cancel and get rid of load and fire here? I imagine with the emergency controller we'd only ever really want to cancel the shooter incase someone's hand was there or something.